tests/test_project_report.py

```diff
@@ -117,11 +134,12 @@ def _resolve_next(o):
 
 fake_process_templates = {
     'a_project_name': {'nb_processes': 1, 'date': d, 'finished': 'Yes', 'NC': 'NC12: Description of major issue'},
-    'hmix999': {'nb_processes': 4, 'date': d, 'finished': 'Yes', 'NC': cycle(['NA', 'NA', 'NC25: Description minor issue', 'NA'])},
+    'hmix999': {'nb_processes': 3, 'date': d, 'finished': 'Yes', 'NC': cycle(['NA', 'NA', 'NC25: Description minor issue', 'NA'])},
```

Comment:
Just out of interest: what does nb_processes represent, and what is the significance of reducing the value from 4 to 3?

Reply:
This parameters change the number of time the test project was released. 
It does not make much of a difference except visually on the report.

Reply:
@tcezard Sorry, I'm not sure I follow. What do you mean by the number of times the test project was released?

Reply:
This generate Mock LIMS steps that represent the data release steps. The `nb_processes` parameter changes the number of such mock steps.
It influences a table on the project report that describes the different releases of the Mocked project.
For the purpose of testing it does not matter if it is 3 or 50. It only influences how long this table is in the report.